### PR TITLE
Admin adds actors feature

### PR DESCRIPTION
**Pull request by ecustovic**
- enabled the main CRUD operations for actors
- after checking if the admin is logged, in the nav bar is shown the Actors tab
- after clicking on the Actors it is shown the actor index page
- for the image field it is enabled to upload the image using Active Storage 
- the gender field it's made as an enum and on the view is shown as radio button
- created the rake task to upgrade seeds for uploading initial images for every single user and also movie because there wasn't possible to upload attachment directly

### CODE REVIEW

File: app/controllers/actors_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+class ActorsController < ApplicationController
```
Comment (dixpac) [inline app/controllers/actors_controller.rb]: In this controller you need to have safe check so non admins can't hit this endpoint. Could be done with controller `before_action :reject_if_not_admin`

Comment (dixpac) [reply app/controllers/actors_controller.rb]: and then you need to test that on the integration tests

File: app/controllers/actors_controller.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  def new
+    @actor = Actor.new
+  end
+
+  def edit
+  end
+
+  def create
+    @actor = Actor.new(actor_params)
+    if @actor.save
```
Comment (dixpac) [inline app/controllers/actors_controller.rb]: Leave default rails `respond_to do |format|..end` here

File: app/controllers/actors_controller.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      redirect_to @actor, notice: "Actor was successfully created."
+    else
+      render :new
+    end
+  end
+
+  def update
+    if @actor.update(actor_params)
```
Comment (dixpac) [inline app/controllers/actors_controller.rb]: Leave default rails `respond_to do |format|..end` here

File: app/controllers/actors_controller.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      redirect_to @actor, notice: "Actor was successfully updated."
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @actor.destroy
```
Comment (dixpac) [inline app/controllers/actors_controller.rb]: Leave default rails `respond_to do |format|..end` here

File: app/views/actors/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+  <div class="field">
+    <%= form.label :birth_date %>
+    <%= form.date_field :birth_date %>
+  </div>
+
+  <div class="field" id="gender">
+    <%= form.label :gender %>
+    <div>
+    <%= form.radio_button :gender, :male %> Male
```
Comment (dixpac) [inline app/views/actors/_form.html.erb]: Can we use https://apidock.com/rails/v4.0.2/ActionView/Helpers/FormOptionsHelper/collection_radio_buttons

File: app/views/actors/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        <th colspan="2"></th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @actors.each do |actor| %>
+          <tr>
+            <td>
+              <% if actor.image.attached? %>
+                <%= image_tag actor.image %>
```
Comment (dixpac) [inline app/views/actors/index.html.erb]: You can move this inside the `ActorsHelper` so you don't clutter views
```ruby
module ActorsHelper
  def actor_avatar(actor)
      if...
  end
end
```

File: app/views/actors/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+            <td><%= link_to 'Edit', edit_actor_path(actor), class: "btn btn-primary" %></td>
+            <td><%= link_to 'Destroy', actor, class: "btn btn-primary", method: :delete, data: { confirm: 'Are you sure?' } %></td>
+          </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <br>
+
+  <% if current_user_admin? %>
```
Comment (dixpac) [inline app/views/actors/index.html.erb]: Typo here...also this means that you didn't cover this edge case inisde the tests

File: app/views/actors/show.html.erb [hunk 8]
```diff
@@ -0,0 +1,25 @@
+<p>
+  <% if @actor.image.attached? %>
```
Comment (dixpac) [inline app/views/actors/show.html.erb]: Same helper method as above can be reused here

File: test/models/actor_test.rb [hunk 9]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ActorTest < ActiveSupport::TestCase
+  # test "the truth" do
```
Comment (dixpac) [inline test/models/actor_test.rb]: You need to test that there is a valid fixture as in other models

File: db/migrate/20191129082950_create_actors.rb [hunk 10]
```diff
@@ -0,0 +1,12 @@
+class CreateActors < ActiveRecord::Migration[6.0]
+  def change
+    create_table :actors do |t|
+      t.string :name
```
Comment (dixpac) [inline db/migrate/20191129082950_create_actors.rb]: It is good idea to add database constraints/validations also, so someone can't enter invalid data directly inside the db, for example `t.string :name, null: false`
